packages/storage/test/service.test.ts

```diff
@@ -45,19 +46,19 @@ describe('Firebase Storage > Service', () => {
       const error = testShared.assertThrows(() => {
         service.ref('gs://bucket/object');
       }, 'storage/invalid-argument');
-      assert.match(error.message, /refFromURL/);
+      error && assert.match(error.message, /refFromURL/);
```

Comment:
Can you see if this is really necessary? assertThrows should always return an error and if it doesn't, the test would pass when it shouldn't.

Reply:
Changing assertThrows back to assuming it always returns an error (instead of allowing it to return null), and should fail a test first if not? Needed some more `@ts-ignore` tags to make it work.